### PR TITLE
Enabled WS ping Timeout

### PR DESCRIPTION
**Pull request by m0hit-aggarwal**

### CODE REVIEW

File: enterprise_gateway/_version.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-__version__ = '2.1.1.dev4'
+__version__ = '2.1.1.dev5'
```
Comment (amangarg96) [inline enterprise_gateway/_version.py]: We are building on top of EG 2.1.1
Can we change the version to 2.1.2dev5?

Comment (vikasgarg1996) [reply enterprise_gateway/_version.py]: Then it will be 2.1.2.dev1 right :P

**Review by amangarg96 — APPROVED** [review]
LGTM

**Review by vikasgarg1996 — APPROVED** [review]
LGTM

**Pull request merged**